backends/mysql.go

```diff
@@ -215,63 +215,63 @@ func NewMysql(authOpts map[string]string, logLevel log.Level, hasher hashing.Has
 }
 
 //GetUser checks that the username exists and the given password hashes to the same password.
-func (o Mysql) GetUser(username, password, clientid string) bool {
+func (o Mysql) GetUser(username, password, clientid string) (bool, error) {
 
 	var pwHash sql.NullString
 	err := o.DB.Get(&pwHash, o.UserQuery, username)
 
 	if err != nil {
 		log.Debugf("MySql get user error: %s", err)
-		return false
+		return false, err
 	}
 
 	if !pwHash.Valid {
 		log.Debugf("MySql get user error: user %s not found", username)
```

Comment:
Again, not on you, but this leaks. OTOH, this one is only in debug level, so maybe not that bad.